staramr/blast/BlastHandler.py

```diff
@@ -200,7 +201,13 @@ def _launch_blast(self, query, db, output) -> None:
         if stderr:
             raise Exception("error with [" + str(blastn_command) + "], stderr=" + stderr)
 
-    def _make_blast_db(self, path: str) -> None:
+    def _make_blast_db(self, path: str, file: str) -> None:
         command = ['makeblastdb', '-in', path, '-dbtype', 'nucl', '-parse_seqids']
         logger.debug(' '.join(command))
-        subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
+        try:
+            subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
+        except subprocess.CalledProcessError as e:
```

Comment:
It looks like no exception is thrown for `makeblastdb` in case of a duplicate sequence.  Also, the `CalledProcessError` exception is thrown whenever the subprocessess (in this case, the command `makeblastdb`) returns a non-zero exit code, which won't necessarily be related to duplicate sequence IDs.

Was there a specific file that was causing this type of exception to be raised?

You could keep this in, but I would change the text from **file {} contains duplicate sequence ID {}** to **Could not run makeblastdb on file {}, error {}** to keep it more generic.

Reply:
Also, please wrap the exception with the cause, that is:

```python
raise Exception('Message',e)
```

Reply:
Yeah there was, it was filed involving `fna` or `fasta` files that contains duplicates where it raised the issue in the `makeblastdb`

Reply:
Resolved this in commit f1e29f2